Magento_Theme/templates/root.phtml

```diff
@@ -1,69 +1,101 @@
+            href="<?= $this->getViewFileUrl('fonts/work-sans-regular.woff2') ?>"
             as="font"
             type="font/woff2"
             crossorigin
         >
 
         <link
             rel="preload"
-            href="<?= /* @noEscape */ $this->getViewFileUrl('fonts/work-sans-medium.woff2') ?>"
+            href="<?= $this->getViewFileUrl('fonts/work-sans-medium.woff2') ?>"
             as="font"
             type="font/woff2"
             crossorigin
         >
 
         <link
             rel="preload"
-            href="<?= /* @noEscape */ $this->getViewFileUrl('fonts/work-sans-bold.woff2') ?>"
+            href="<?= $this->getViewFileUrl('fonts/work-sans-bold.woff2') ?>"
             as="font"
             type="font/woff2"
             crossorigin
         >
 
         <link
             rel="preload"
-            href="<?= /* @noEscape */ $this->getViewFileUrl('fonts/playfair-display-regular.woff2') ?>"
+            href="<?= $this->getViewFileUrl('fonts/playfair-display-regular.woff2') ?>"
             as="font"
             type="font/woff2"
             crossorigin
         >
 
+        <style>
+            @font-face {
+                font-family: 'Work Sans';
+                src: url('<?= $this->getViewFileUrl('fonts/work-sans-regular.woff2') ?>') format('woff2');
+                font-weight: 400;
+                font-style: normal;
+                font-display: block;
+            }
+
+            @font-face {
+                font-family: 'Work Sans';
+                src: url('<?= $this->getViewFileUrl('fonts/work-sans-medium.woff2') ?>') format('woff2');
+                font-weight: 500;
+                font-style: normal;
+                font-display: block;
+            }
+
+            @font-face {
+                font-family: 'Work Sans';
+                src: url('<?= $this->getViewFileUrl('fonts/work-sans-bold.woff2') ?>') format('woff2');
+                font-weight: 700;
+                font-style: normal;
+                font-display: block;
+            }
+
+            @font-face {
+                font-family: 'Playfair Display';
+                src: url('<?= $this->getViewFileUrl('fonts/playfair-display-regular.woff2') ?>') format('woff2');
+                font-weight: 400;
+                font-style: normal;
+                font-display: block;
+            }
+        </style>
+
+        <style>
+            <?php
+                $criticalStyles = $this->assetRepo->createAsset('css/critical.css')->getContent();
+                echo str_replace('../images', $this->getViewFileUrl('images'), $criticalStyles);
+            ?>
+        </style>
+
         <link
-            href="<?= /* @noEscape */ $this->getViewFileUrl('mage/calendar.css') ?>"
-            rel="stylesheet preload"
+            href="<?= $this->getViewFileUrl('css/styles.css') ?>"
+            rel="stylesheet"
             as="style"
+            media="print"
+            onload="this.media='all'"
```

Comment:
weird but with `bin/magento config:set dev/css/use_css_critical_path 1` it doesn't work and I don't see any content :thinking: 
looks like `styles.css` is not applied and critical.css is hiding the content, but when it's set to `0` it works for me


rendered `<link>` with `use_css_critical_path 1`:
`<link rel="stylesheet" media="print" onload="this.onload=null;this.media='print'" href="https://colibri.test/static/frontend/Snowdog/alpaca/en_US/css/styles.css">`

with: `use_css_critical_path 0` (all good)
`<link href="https://colibri.test/static/frontend/Snowdog/alpaca/en_US/css/styles.css" rel="stylesheet" media="all" onload="this.media=&quot;all&quot;">`

Reply:
It's not based on `use_css_critical_path`, and I don't want to use it.

Reply:
I know that it's not based on `onuse_css_critical_path` but it will break styles for a project that uses it